Makefile

```diff
@@ -76,20 +83,14 @@ space := $() $()
 comma := ,
 GOFLAGS := -tags=$(subst $(space),$(comma),$(strip $(BUILTIN_CLOUD_PROVIDERS)))
 
-#
-# If SKIP_TESTS is set, the test target will *not* run `go test`.
-# This is to be able to temporarily work around test failures when doing
-# local development.
-SKIP_TESTS =
```

Comment:
Won't removing this risk breaking CI again, along with restoring an annoyance for devs who don't necessarily want to run the tests on each build?

Reply:
If you see the `test` target is no longer called as part of other build targets. It's explicitly called. So it should be fine imho

Reply:
Ah alright, I missed that, sorry.